Setup/Product.wxs

```diff
@@ -198,6 +203,10 @@
                     <RegistryValue Value="&quot;[INSTALLDIR]GitExtensions.exe&quot; %1" Type="string" />
                 </RegistryKey>
             </Component>
+            <Component Id="VS2015_VSIX" Guid="*">
+                <VSExtension:VsixPackage File="GitExtensionsVSIX.vsix" PackageId="GitExtensions..3166f76a-2b07-410a-a72a-509d6a2d146c" Target="community" TargetVersion="14.0" Vital="yes" Permanent="no" VsixInstallerPathProperty="VS2015_VSIX_INSTALLER_PATH" />
+                <File Source="..\GitExtensionsVSIX\bin\Release\GitExtensionsVSIX.vsix" />
```

Comment:
Arkady @KindDragon, how can I get `GitExtensionsVSIX\bin\Release\GitExtensionsVSIX.vsix` file?
I'm trying to make an installer, but finally `MakeInstallers.cmd` fails missing this file. I have no enough knowledge to guess how to get the file appear as I'm not a VS user. Thank you!